src/app/who-did-this/who-did-this.component.scss

```diff
@@ -0,0 +1,35 @@
+.team-container {
+  padding-bottom: 2rem;
+  .header {
+    color: #ee6e73;
+    font-weight: 300;
+  }
+  .center {
```

Comment:
same thing with this class, use helpers of materialize: valign and center-align instead of creating new ones